src/xrpld/app/misc/WasmHostFuncImpl.cpp

```diff
@@ -153,21 +177,19 @@ WasmHostFunctionsImpl::getTxField(SField const& fname)
 Expected<Bytes, HostFunctionError>
 WasmHostFunctionsImpl::getCurrentLedgerObjField(SField const& fname)
 {
-    auto const sle = ctx.view().read(leKey);
-    if (!sle)
-        return Unexpected(HostFunctionError::LEDGER_OBJ_NOT_FOUND);
-    return getAnyFieldData(sle->peekAtPField(fname));
+    auto const sle = getCurrentLedgerObj();
+    if (!sle.has_value())
+        return Unexpected(sle.error());
+    return getAnyFieldData(sle.value()->peekAtPField(fname));
 }
 
 Expected<Bytes, HostFunctionError>
-WasmHostFunctionsImpl::getLedgerObjField(int32_t cacheIdx, SField const& fname)
+WasmHostFunctionsImpl::getLedgerObjField(int32_t idxParam, SField const& fname)
```

Comment:
idxParam is a bad name as everything is "param" here. Better rename them to `idx` and `normalIdx`.  And `getActualCacheIndex` to `normalizeIdx`

Reply:
`normalize` feels confusing to me, because which of 0-indexing and 1-indexing is "normal"?